zebra/zebra_vty.c

```diff
@@ -1330,35 +1330,56 @@ static int do_show_ip_route(struct vty *vty, const char *vrf_name, afi_t afi,
 	return CMD_SUCCESS;
 }
 
-DEFPY (show_ip_nht,
-       show_ip_nht_cmd,
-       "show <ip$ipv4|ipv6$ipv6> <nht|import-check>$type [<A.B.C.D|X:X::X:X>$addr|vrf NAME$vrf_name [<A.B.C.D|X:X::X:X>$addr]|vrf all$vrf_all]",
-       SHOW_STR
-       IP_STR
-       IP6_STR
-       "IP nexthop tracking table\n"
-       "IP import check tracking table\n"
-       "IPv4 Address\n"
-       "IPv6 Address\n"
-       VRF_CMD_HELP_STR
-       "IPv4 Address\n"
-       "IPv6 Address\n"
-       VRF_ALL_CMD_HELP_STR)
+DEFPY(show_ip_nht, show_ip_nht_cmd,
+      "show <ip$ipv4|ipv6$ipv6> <nht|import-check>$type "
+      "[<A.B.C.D|X:X::X:X>$addr|vrf NAME$vrf_name [<A.B.C.D|X:X::X:X>$addr]|vrf all$vrf_all] [json$json]",
+      SHOW_STR IP_STR IP6_STR
```

Comment:
Please don't re-wrap these tokens - I know the tools don't always understand what to do with these blocks, but our convention is: space before the open-paren, and one token per line. I don't mind your wrapping the double-quoted string - that's ok, but not the help text statements.

Reply:
Thanks @mjstapp , clang tool made this alignment . I have addressed your suggested change . Lets hope it will get pass.